generator/src/test/java/com/scottlogic/deg/generator/validators/GenerationConfigValidatorTests.java

```diff
@@ -1,29 +1,52 @@
 package com.scottlogic.deg.generator.validators;
 
+import com.scottlogic.deg.generator.Profile;
 import com.scottlogic.deg.generator.generation.GenerationConfig;
 import com.scottlogic.deg.generator.generation.TestGenerationConfigSource;
+import com.scottlogic.deg.generator.outputs.targets.FileOutputTarget;
+import com.scottlogic.deg.generator.utils.FileUtils;
 import org.junit.Assert;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
+import java.util.ArrayList;
+
 import static org.hamcrest.collection.IsEmptyCollection.empty;
 import static org.hamcrest.core.Is.is;
 import static org.hamcrest.core.IsCollectionContaining.hasItem;
+import static org.mockito.Matchers.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 
 public class GenerationConfigValidatorTests {
-    @Test
-    public void interestingWithNoMaxRowsReturnsValid() {
+
+    private Profile profile;
+    private FileUtils fileUtils=mock(FileUtils.class);
+    private FileOutputTarget outputTarget = mock(FileOutputTarget.class);
+    private TestGenerationConfigSource mockConfigSource = mock(TestGenerationConfigSource.class);
+    private GenerationConfig config = new GenerationConfig(
+        new TestGenerationConfigSource(
+            GenerationConfig.DataGenerationType.INTERESTING,
+            GenerationConfig.TreeWalkerType.REDUCTIVE,
+            GenerationConfig.CombinationStrategyType.EXHAUSTIVE
+        )
+    );
+    private GenerationConfigValidator validator;
+
+    @BeforeEach
+    void setup() {
         //Arrange
-        GenerationConfig config = new GenerationConfig(
-            new TestGenerationConfigSource(
-                GenerationConfig.DataGenerationType.INTERESTING,
-                GenerationConfig.TreeWalkerType.REDUCTIVE,
-                GenerationConfig.CombinationStrategyType.EXHAUSTIVE
-                )
-        );
-        GenerationConfigValidator validator = new GenerationConfigValidator();
+        validator = new GenerationConfigValidator(mockConfigSource, outputTarget, fileUtils);
+        when(fileUtils.isDirectory(anyObject())).thenReturn(false);
```

Comment:
The mock could be more specific, you know what you'll be passing to this based on the output target used to new up the class

Reply:
yes, it should be the object. updated now.